services/apps/src/apps/service/workspace.clj

```diff
@@ -1,9 +1,11 @@
 (ns apps.service.workspace
   (:use [korma.db :only [transaction]])
-  (:require [apps.persistence.workspace :as wp]))
+  (:require [apps.clients.iplant-groups :as iplant-groups]
+            [apps.persistence.workspace :as wp]))
 
 (defn get-workspace
-  [{:keys [username]}]
+  [{short-username :shortUsername :keys [username]}]
+  (iplant-groups/add-de-user short-username)
```

Comment:
Cool. I'll create a Jira issue to ensure that this doesn't fall through the cracks.